packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/document/DocumentAsset.java

```diff
@@ -35,7 +35,7 @@ public class DocumentAsset extends TerariumAsset {
 	private String userId;
 
 	@TSOptional
-	@JsonAlias("file_names")
```

Comment:
Sorry do not do that, leave it as is and we will fix it properly in es with @kbirk